src/main/scala/org/broadinstitute/dsde/workbench/sam/api/AdminRoutes.scala

```diff
@@ -187,36 +192,72 @@ trait AdminRoutes extends SecurityDirectives with SamRequestContextDirectives wi
       }
     }
 
-  def adminResourceTypesRoutes(user: SamUser, samRequestContext: SamRequestContext): server.Route =
-    pathPrefix("resourceTypes" / Segment / "policies") { resourceTypeNameToAdminister =>
-      asSamSuperAdmin(user) {
-        withNonAdminResourceType(ResourceTypeName(resourceTypeNameToAdminister)) { resourceTypeToAdminister =>
-          val resource = FullyQualifiedResourceId(resourceTypeAdminName, ResourceId(resourceTypeToAdminister.name.value))
-          pathEndOrSingleSlash {
-            getWithTelemetry(samRequestContext, resourceTypeParam(resourceTypeToAdminister)) {
+  def adminGroupsRoutes(user: SamUser, samRequestContext: SamRequestContext): server.Route =
+    path("groups" / Segment / "supportSummary") { groupName =>
+      pathEndOrSingleSlash {
+        withResourceType(ManagedGroupService.managedGroupTypeName) { managedGroupType =>
+          val groupId = FullyQualifiedResourceId(ManagedGroupService.managedGroupTypeName, ResourceId(groupName))
+          getWithTelemetry(samRequestContext, groupIdParam(groupId)) {
+            requireAdminResourceAction(SamResourceActions.adminReadSummaryInformation, managedGroupType, user, samRequestContext) {
               complete {
-                resourceService
-                  .listResourcePolicies(resource, samRequestContext)
-                  .map(response => OK -> response.toSet)
+                managedGroupService.loadManagedGroupSupportSummary(groupId.resourceId, samRequestContext).flatMap {
+                  case Some(response) => IO.pure(StatusCodes.OK -> response)
+                  case None => IO.raiseError(new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.NotFound, "group not found")))
+                }
               }
             }
-          } ~
-          pathPrefix(Segment) { policyName =>
-            val policyId = FullyQualifiedPolicyId(resource, AccessPolicyName(policyName))
+          }
+        }
+      }
+    }
+
+  def adminResourceTypesRoutes(user: SamUser, samRequestContext: SamRequestContext): server.Route =
+    pathPrefix("resourceTypes" / Segment) { resourceTypeNameToAdminister =>
```

Comment:
Similar comment to above do we need to wrap this new endpoint in `asWorkbenchAdmin`? Or is the idea that the user making the call just needs to be on the broad network and have the appropriate role?

Reply:
just needs to be on the broad network and have the appropriate role